android/project.properties

```diff
@@ -12,3 +12,4 @@
 
```

Comment:
Why appcompat? I don't mind introducing appcompat if backwards compatibility is considered, but it should be via gradle and not project.properties. 

This path will be broken on other peoples machines.